spec/manifolds/cli_spec.rb

```diff
@@ -3,86 +3,107 @@
 require_relative "../../lib/manifolds/cli"
 require "fileutils"
 require "logger"
+require "fakefs/spec_helpers"
 
 RSpec.describe Manifolds::CLI do
+  include FakeFS::SpecHelpers
+
   let(:project_name) { "commerce" }
   let(:sub_project_name) { "Pages" }
   let(:null_logger) { Logger.new(File::NULL) }
-  let(:bq_service) { instance_double("Manifolds::Services::BigQueryService", "commerce") }
+
+  before do
+    templates_dir = File.expand_path("../../lib/manifolds/templates", __dir__)
+    FileUtils.mkdir_p(templates_dir)
+
+    File.write(File.join(templates_dir, "config_template.yml"), "vectors:\nmetrics:")
+    File.write(File.join(templates_dir, "vector_template.yml"), "attributes:")
+  end
 
   describe "#init" do
     subject(:cli) { described_class.new(logger: null_logger) }
 
-    before do
-      allow(FileUtils).to receive(:mkdir_p)
-      allow(File).to receive(:open)
-      cli.init(project_name)
-    end
+    context "when initializing a new project" do
+      before { cli.init(project_name) }
 
-    it "creates the projects directory" do
-      expect(FileUtils).to have_received(:mkdir_p).with("./#{project_name}/projects")
+      it "creates a 'projects' directory" do
+        expect(Dir.exist?(File.join(Dir.pwd, project_name, "projects"))).to be true
```

Comment:
Unless you are implementing `Dir.exist?` and you absolutely require it to return a boolean, use `be_truthy` instead of `be true`.

Of course, as I've said elsewhere, using `Pathname` allows for much more readable tests:

```ruby
expect(Pathname.pwd.join(project_name, "projects")).to be_directory
```